### PR TITLE
Adding Classic ROSA 4.16 jobs to prow

### PR DESCRIPTION
**Pull request by vishnuchalla**
### Description
Adding Classic ROSA 4.16 jobs following the rules [here](https://docs.google.com/document/d/1NbXwfZwSkMdftuKW1Ob705rQk-XHLtPtP1d1vuqF9Vo/edit#heading=h.it8c4239uo7a)

### Testing
Will test and verify through this PR rehearsals itself.

### CODE REVIEW

**Comment by vishnuchalla** [comment]
Let me know if we want to add `node-density-cni` job for M and L size tests too.

**Comment by vishnuchalla** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.16-nightly-x86-data-path-9nodes

**Comment by vishnuchalla** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.16-nightly-x86-control-plane-24nodes

File: ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml [hunk 1]
```diff
@@ -5572,6 +5572,333 @@ periodics:
...
+    org: openshift-qe
+    repo: ocp-qe-perfscale-ci
+  labels:
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws-perfscale
+    ci-operator.openshift.io/variant: rosa-4.16-nightly-x86
+    ci.openshift.io/generator: prowgen
+    job-release: "4.16"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.16-nightly-x86-control-plane-120nodes
```
Comment (afcollins) [inline ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml:5593]: Please add slack reporter_config

Comment (vishnuchalla) [reply ci-operator/jobs/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main-periodics.yaml]: Done. Added and updated.

**Comment by svetsa-rh** [comment]
waiting for Andrew to check and confirm crons, looks good otherwise

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.16-nightly-x86.yaml [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+      MUST_GATHER_TIMEOUT: 90m
+      OPENSHIFT_VERSION: "4.16"
+      PROFILE_TYPE: reporting
+      REPLICAS: "249"
+    test:
+    - chain: openshift-qe-cluster-density-v2
+    workflow: rosa-aws-sts
+  timeout: 8h0m0s
+- as: control-plane-120nodes
+  cron: 0 2 11,25 * *
```
Comment (afcollins) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.16-nightly-x86.yaml:56]: Suggest: `"0 6 4,18 * *"`

Currently, same hour as aws 4.17.

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.16-nightly-x86.yaml [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+      MULTI_AZ: "true"
+      OPENSHIFT_VERSION: "4.16"
+      READY_WAIT_TIMEOUT: 30m
+      REPLICAS: "24"
+    test:
+    - chain: openshift-qe-control-plane
+    workflow: rosa-aws-sts
+  timeout: 5h0m0s
+- as: data-path-9nodes
+  cron: 0 6 1,15 * *
```
Comment (afcollins) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.16-nightly-x86.yaml:93]: Suggest: `"0 14 8,22 * *"`

**Review by afcollins — APPROVED** [review]
Thanks!

**Pull request merged**